core/mondoo-azure-security.mql.yaml

```diff
@@ -564,17 +556,16 @@ queries:
     impact: 80
     mql: |
       azure.subscription.network.securityGroups {
-        securityRules.where( properties['access'] == 'Allow' && properties['direction'] == 'Inbound' && properties['protocol'] == /TCP|\*/ && properties['sourceAddressPrefix'] == /\*|0\.0\.0\.0|<nw>\/0|\/0|internet|any/ )
-        } {
-          securityRules {
-            properties["destinationPortRange"] != "*"
-            properties["destinationPortRange"] != 3389
-            if (properties["destinationPortRange"].split("-").length > 1) {
-              a = properties["destinationPortRange"].split("-")[0] < 3389 && properties["destinationPortRange"].split("-")[1] < 3389
-              b = properties["destinationPortRange"].split("-")[0] > 3389 && properties["destinationPortRange"].split("-")[1] > 3389
-              a || b
+        securityRules.where(properties.access == 'Allow' && direction == 'Inbound' && properties.protocol == /TCP|\*/ && properties.sourceAddressPrefix == /\*|0\.0\.0\.0|<nw>\/0|\/0|internet|any/)
```

Comment:
Same as above.

Reply:
Thank you for the suggestion. However, it appears that it doesn't effectively address all the test scenarios we have. I appreciate your input and look forward to exploring alternative solutions or Keeping the Original one that might be more comprehensive.